apps/research-service/src/site-polygons/site-polygon-query.builder.ts

```diff
@@ -0,0 +1,161 @@
+import { Attributes, Filterable, FindOptions, IncludeOptions, literal, Op, WhereOptions } from "sequelize";
+import {
+  IndicatorOutputFieldMonitoring,
+  IndicatorOutputHectares,
+  IndicatorOutputMsuCarbon,
+  IndicatorOutputTreeCount,
+  IndicatorOutputTreeCover,
+  IndicatorOutputTreeCoverLoss,
+  PolygonGeometry,
+  Project,
+  Site,
+  SitePolygon,
+  SiteReport,
+  TreeSpecies
+} from "@terramatch-microservices/database/entities";
+import { IndicatorSlug, PolygonStatus } from "@terramatch-microservices/database/constants";
+import { uniq } from "lodash";
+import { BadRequestException } from "@nestjs/common";
+
+type IndicatorModelClass =
+  | typeof IndicatorOutputTreeCover
+  | typeof IndicatorOutputTreeCoverLoss
+  | typeof IndicatorOutputHectares
+  | typeof IndicatorOutputTreeCount
+  | typeof IndicatorOutputFieldMonitoring
+  | typeof IndicatorOutputMsuCarbon;
+
+export const INDICATOR_MODEL_CLASSES: { [Slug in IndicatorSlug]: IndicatorModelClass } = {
+  treeCover: IndicatorOutputTreeCover,
+  treeCoverLoss: IndicatorOutputTreeCoverLoss,
+  treeCoverLossFires: IndicatorOutputTreeCoverLoss,
+  restorationByEcoRegion: IndicatorOutputHectares,
+  restorationByStrategy: IndicatorOutputHectares,
+  restorationByLandUse: IndicatorOutputHectares,
+  treeCount: IndicatorOutputTreeCount,
+  earlyTreeVerification: IndicatorOutputTreeCount,
+  fieldMonitoring: IndicatorOutputFieldMonitoring,
+  msuCarbon: IndicatorOutputMsuCarbon
+};
+
+const INDICATOR_EXCLUDE_COLUMNS = ["id", "sitePolygonId", "createdAt", "updatedAt", "deletedAt"];
+
+export class SitePolygonQueryBuilder {
```

Comment:
This class hasn't had any logic changes in this ticket; it just got moved out of the service class.